src/gui/guiEditBoxWithScrollbar.cpp

```diff
@@ -620,6 +620,17 @@ void GUIEditBoxWithScrollBar::createVScrollBar()
 	if (Environment)
 		skin = Environment->getSkin();
 
+	s32 fontHeight = 1;
+
+	if (m_override_font) {
+		fontHeight = m_override_font->getDimension(L"").Height;
+	} else {
+		IGUIFont *font;
+		if (skin && (font = skin->getFont())) {
+			fontHeight = font->getDimension(L"").Height;
```

Comment:
technically an empty string has no dimensions but if it works elsewhere...

Reply:
Normally `Ay` is used in Minetest to determine the maximal line height

Reply:
Changed to `Ay` now. It both gives 20 on my machine. It was empty string to match what intlGUIEditBox did:
https://github.com/minetest/minetest/blob/285ba74723695c4b51192dac0e1e17c5d8f880db/src/gui/intlGUIEditBox.cpp#L603-L613

Reply:
It makes absolutely no difference.  The height returned is always the same.  Ideally, Irrlicht would have a `getHeight()` and `getWidth(string)`, but Irrlicht is far from ideal.